tools/pika_exporter/Makefile

```diff
@@ -65,8 +66,10 @@ all: build
 build: deps
 ifeq ($(UNAME), Linux)
```

Comment:
Do you need to consider the situation of Linux and arm architecture?

Reply:
done